client/src/components/Draftail/blocks/MediaBlock.scss

```diff
@@ -25,7 +25,7 @@
 
     @mixin invalid-image-fallback {
         min-width: 256px;
-        min-height: 100px;
+        min-height: 25px;
```

Comment:
Checking what this looks like in the editor, I realise that 25px would be smaller than the image block’s top-right icon and not look right 😞